.eslintignore

```diff
@@ -0,0 +1,15 @@
+lib/ace/mode/css/*
+lib/ace/mode/coffee/*
+lib/ace/mode/html/*
+lib/ace/mode/javascript/*
+lib/ace/mode/json/*
+lib/ace/mode/lua/*
+lib/ace/mode/php/*
+lib/ace/mode/xml/*
+lib/ace/mode/xquery/*
+lib/ace/mode/xquery.js
+lib/ace/test/asyncjs/
+lib/ace/lib/es5-shim.js
+lib/ace/keyboard/vim*.js
```

Comment:
Should this also have something like the following?

```
+doc/*
+build/**
+tool/templates/*
```

Running with that I still see 92 errors